app/services/funnel/doc_auth/register_step_from_analytics_view_event.rb

```diff
@@ -10,7 +10,19 @@ class RegisterStepFromAnalyticsViewEvent
 
       def self.call(user_id, issuer, event, result)
```

Comment:
this is the file I missed entirely last time 😬  as far as I can tell it's just these 2! shouldn't be a mystery 3rd one